benchmark/benchwrapper/main.py

```diff
@@ -0,0 +1,157 @@
+# Copyright 2022 Google, Inc.
```

Comment:
There are several different copyrights in this PR. Some are for Google Inc 2022, some for Google LLC 2019 and so on. Should be checked for which is right.

Reply:
I think it's best if we stick to 1 format for the entire projec. **Do not modify any existing files**.
Just check which is the common format in this project and across projects (search on VS code/ Github) for copyright word. Use the format which is commonly used.

Reply:
I'll change this to "Copyright 2022 Google LLC" to make it consistent with the other files in the repository.

Not changing the copyright in spanner.proto file as it has been copied over from another internal repository.

Reply:
Done.